MaxText/configs/base.yml

```diff
@@ -281,6 +281,10 @@ inference_microbenchmark_stages: "prefill,generate"
 inference_microbenchmark_loop_iters: 10
 inference_microbenchmark_log_file_path: ""
 
+# Metrics
```

Comment:
There are other metric related fields already define here, could we change the comment header to Maxengine metrics?